src/diff-text/index.jsx

```diff
@@ -18,7 +18,7 @@ const DiffText = ({
   currentLabel = 'Current',
   proposedLabel = 'Proposed'
 }) => {
-  const diff = jsdiff.diffChars(oldValue, newValue);
+  const diff = jsdiff.diffWords(oldValue, newValue);
```

Comment:
It makes no difference if we `import { difWords } from 'diff';` in terms of bundle size, does it?

Reply:
your RIGHT - updated

Reply:
I don't know what you've updated, but meh.